index.html

```diff
@@ -0,0 +1,44 @@
+<link rel="stylesheet" href="stylesTributePage.css">
+
+<style>
+   body {
+      background-color: rgb(90, 0, 80);
+      color: white;
+      font-family: calibri;
+   }
+</style>
+
+<html lang="en">
+
+<head>
+   <title>Tribute Page</title>
+</head>
+
+<body>
+
+   <div id="main">
+      <h1 id="title">Sevagoth - The slayer</h1>
+
+      <div id="img-div">
+         <a id="tribute-link" target="_blank" href="https://warframe.fandom.com/wiki/Sevagoth">
+            <img id="image"
+               src="https://www.ggrecon.com/media/x4tlnglf/warframe-sevagoth-2.jpg?mode=crop&width=682&quality=80&format=webp"
+               alt="Super duper cool Sevagoth image" width="100%"></a>
```

Comment:
Good job in linking the image to an external link, however, I would suggest removing the "width" from the index.html file. Adding a "width" property in HTML and in CSS will cause conflict between the two. Kindly apply this to ensure that your image does what you intend it to do.